integration-test/common/src/test/java/com/tencent/rss/test/QuorumTest.java

```diff
@@ -94,13 +96,20 @@ public static void initCluster() throws Exception {
     shuffleServers.add(createServer(0));
     shuffleServers.add(createServer(1));
     shuffleServers.add(createServer(2));
+    shuffleServers.add(createServer(3));
+    shuffleServers.add(createServer(4));
+
 
     shuffleServerInfo0 =
-      new ShuffleServerInfo("127.0.0.1-20001", shuffleServers.get(0).getIp(), SHUFFLE_SERVER_PORT);
+      new ShuffleServerInfo("127.0.0.1-20001", shuffleServers.get(0).getIp(), SHUFFLE_SERVER_PORT + 0);
     shuffleServerInfo1 =
       new ShuffleServerInfo("127.0.0.1-20002", shuffleServers.get(1).getIp(), SHUFFLE_SERVER_PORT + 1);
     shuffleServerInfo2 =
-      new ShuffleServerInfo("127.0.0.1-20003", shuffleServers.get(1).getIp(), SHUFFLE_SERVER_PORT + 2);
+      new ShuffleServerInfo("127.0.0.1-20003", shuffleServers.get(2).getIp(), SHUFFLE_SERVER_PORT + 2);
+    shuffleServerInfo3 =
+      new ShuffleServerInfo("127.0.0.1-20004", shuffleServers.get(3).getIp(), SHUFFLE_SERVER_PORT + 3);
+    shuffleServerInfo4 =
+      new ShuffleServerInfo("127.0.0.1-20005", shuffleServers.get(4).getIp(), SHUFFLE_SERVER_PORT + 4);
```

Comment:
add more case,such as one partition report success, one partition report fail, but total success count reach , and cherry-pick this patch to branch 0.4.0.

Reply:
ok